packages/calcite-components/src/components/radio-button/radio-button.e2e.ts

```diff
@@ -83,6 +83,51 @@ describe("calcite-radio-button", () => {
     expect(value).toBe("third");
   });
 
+  it("skips disabled radio-buttons", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
```

Comment:
Can you use the `html` helper so Prettier can format this HTML (see [example](https://github.com/Esri/calcite-design-system/blob/972f8380da13039a4aacdc12b58ffbe8031de634/packages/calcite-components/src/components/radio-button/radio-button.e2e.ts#L93))?